packages/circuits/README.md

```diff
@@ -1,5 +1,53 @@
-# Notes 
+# @anon-aadhaar/circuits
 
-Those circuit copy from [zk-email-verify](https://github.com/zkemail/zk-email-verify/tree/bd4e2412735c09499df93d17cf3180b65711483f).
+These package contains all the circuits and setup that are the core of the Anon Aadhaar protocol.
 
-We built qr verify from circuits built by zkemail. Thank zkemail for awesome work!!!
\ No newline at end of file
+## Structure
+
+The main logic of the protocol is contained in:
+
+- `src/aadhaar-verifier.circom`: This is the main circuit that encapsulates the core logic of the Anon Aadhaar protocol. It handles the verification process and ensures the integrity and confidentiality of Aadhaar data.
+
+- `src/extractor.circom`: This circuit is responsible for securely extracting and processing relevant information from Aadhaar data.
+
+The circuits in charge of the Sha256 and RSA verification are adapted from the [zk-email](https://github.com/zkemail/zk-email-verify/tree/bd4e2412735c09499df93d17cf3180b65711483f) project. We extend our gratitude to zkemail for their exceptional work in this domain.
+
+## 📜 Installation and Setup
+
+### Installation
+
+```bash
+yarn dev-install
+```
+
+### Run the development trusted setup
+
+```bash
+yarn dev-setup
+```
+
+### Run the development trusted setup for the Aadhaar QR code data
```

Comment:
This script to is to test against v1 aadhaar test data right? I think the headlinf can include "test" or "v1"